lib/phoenix_ecto/plugs/check_repo_ready.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Phoenix.Ecto.Plugs.CheckRepoReady do
```

Comment:
I would remove the `Plugs` namespace. Especially because people will know this exists by looking at their apps, which will have this:

    plug Phoenix.Ecto.Plugs.CheckRepoReady

and by then it is obvious it is a plug. I think we should also pluralize, so I would go with `Phoenix.Ecto.CheckReposReady`.

Reply:
@chrismccord suggested `Phoenix.Ecto.CheckRepoStatus` (singular and with Status).

Reply:
[fixed here](https://github.com/phoenixframework/phoenix_ecto/compare/693fbe89c367841069104dcf2070e1e40f9f0028..d85122ccded004efe2b01f32d66e64073c16766f)